lib/openstax/aws/stack.rb

```diff
@@ -1,141 +1,9 @@
 require 'json'
+require_relative 'stack_event'
+require_relative 'stack_status'
```

Comment:
I think you can move these inner class requires to the end of the `openstax_aws.rb` file where the other ones are?